Neo.Gui.ViewModels/Accounts/CreateLockAccountViewModel.cs

```diff
@@ -2,69 +2,39 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
-using System.Windows.Input;
 
 using GalaSoft.MvvmLight;
 using GalaSoft.MvvmLight.Command;
 
-using Neo.SmartContract;
-using Neo.VM;
-using Neo.Wallets;
-
-using Neo.Gui.Globalization.Resources;
+using Neo.Cryptography.ECC;
 
 using Neo.Gui.Base.Controllers.Interfaces;
 using Neo.Gui.Base.Dialogs.Interfaces;
 using Neo.Gui.Base.Dialogs.Results.Wallets;
 using Neo.Gui.Base.Managers.Interfaces;
+using Neo.Gui.Base.MVVM;
 
 namespace Neo.Gui.ViewModels.Accounts
 {
-    public class CreateLockAccountViewModel : ViewModelBase, IDialogViewModel<CreateLockAccountDialogResult>
+    public class CreateLockAccountViewModel : ViewModelBase, IDialogViewModel<CreateLockAccountDialogResult>, ILoadable
     {
+        #region Private Fields 
         private const int HoursInDay = 24;
         private const int MinutesInHour = 60;
 
         private readonly IDialogManager dialogManager;
         private readonly IWalletController walletController;
 
-        private KeyPair selectedKeyPair;
+        private ECPoint selectedKeyPair;
         private DateTime unlockDate;
         private int unlockHour;
         private int unlockMinute;
+        #endregion
 
-        public CreateLockAccountViewModel(
-            IDialogManager dialogManager,
-            IWalletController walletController)
-        {
-            this.dialogManager = dialogManager;
-            this.walletController = walletController;
-
-            this.KeyPairs = new ObservableCollection<KeyPair>(
-                walletController.GetStandardAccounts()
-                    .Select(p => p.GetKey()).ToArray());
-
-            this.Hours = new List<int>();
-
-            this.Hours = Enumerable.Range(0, HoursInDay).ToList();
-
-            this.Minutes = Enumerable.Range(0, MinutesInHour).ToList();
-
-            var now = DateTime.UtcNow;
-
-            this.MinimumDate = now.Date;
-            this.UnlockDate = now;
-
-            // Set unlock time
-            var time = now.TimeOfDay;
-
-            this.UnlockHour = time.Hours;
-            this.UnlockMinute = time.Minutes;
-        }
-
-        public ObservableCollection<KeyPair> KeyPairs { get; }
+        #region Public Properties 
+        public ObservableCollection<ECPoint> KeyPairs { get; }      // TODO: this is not KeyPairs anymore but a list of PublicKeys. Need to be checked with jlgaffney
```

Comment:
I've got no problem with just using the public keys, the private key is not needed. Maybe rename the `KeyPairs` property to `PublicKeys` though, so it is clearer what the property is for